nsv13/code/modules/overmap/fighters/_fighters.dm

```diff
@@ -403,6 +403,35 @@ Been a mess since 2018, we'll fix it someday (probably)
 						/obj/item/fighter_component/battery,
 						/obj/item/fighter_component/primary/cannon)
 
+/obj/structure/overmap/small_craft/combat/solgov
+	name = "Peregrine class attack fighter"
+	desc = "A Peregrine class attack fighter, solgov's only premiere fighter, mounting minature capital grade phasers and a tiny shield generator."
+	icon = 'nsv13/icons/overmap/new/solgov/playablefighter.dmi'
+	armor = list("melee" = 60, "bullet" = 60, "laser" = 60, "energy" = 30, "bomb" = 30, "bio" = 100, "rad" = 90, "fire" = 90, "acid" = 80, "overmap_light" = 5, "overmap_medium" = 0, "overmap_heavy" = 10) // no armor to speak of, tissue paper
+	sprite_size = 32
+	damage_states = FALSE //temp
+	max_integrity = 25 //shields.
+	max_angular_acceleration = 200
+	speed_limit = 10
+	pixel_w = -16
+	pixel_z = -20
+	components = list(/obj/item/fighter_component/fuel_tank,
+						/obj/item/fighter_component/avionics,
+						/obj/item/fighter_component/apu,
+						/obj/item/fighter_component/targeting_sensor,
+						/obj/item/fighter_component/engine,
+						/obj/item/fighter_component/countermeasure_dispenser,
+						/obj/item/fighter_component/oxygenator,
+						/obj/item/fighter_component/canopy,
+						/obj/item/fighter_component/docking_computer,
+						/obj/item/fighter_component/battery,
+						/obj/item/fighter_component/primary/laser)   // no armor because >=3, you can still install it though because this thing is made of tissue paper
+
+/obj/structure/overmap/small_craft/combat/solgov/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/overmap_shields, 125, 125, 15) //inital integrity, max integrity, and recharge rate. bound to change most likely
```

Comment:
Yeah. No. Fighters are intentionally fragile for gameplay reasons. The solgov fighters that have shields do so because they're part hunter-killer fleets designed to kill the player ship when the IFF is flipped.

Reply:
shields don't benefit from armor, meaning that despite having 125 hp, it only takes a couple pdc rounds to completely disable it, or basically anything else, and the hull is next to non-existent. I did some playtesting, and if you aren't on the top of your game, you die *faster* in this fighter than in a regular one

Reply:
player fighters with shields on them are a hard no at this time after much discussion within the team. if anyone else wishes to further elaborate they may, but This is just as a notification

Reply:
I'll refrain from mapping them anywhere in that case